core/src/main/java/bio/terra/pearl/core/service/study/StudyEnvironmentSurveyService.java

```diff
@@ -22,6 +24,13 @@ public Optional<StudyEnvironmentSurvey> findBySurvey(UUID studyEnvId, UUID surve
         return dao.findBySurvey(studyEnvId, surveyId);
     }
 
+    @Transactional
+    public StudyEnvironmentSurvey deactivate(UUID id) {
```

Comment:
good question.  We do need 'delete' for populating, and I wouldn't be surprised if we also will need it in the near future for some sort of "cleanup" operation that study staff will want to do:  "I created 10 versions of this survey while I was futzing around, please destroy all but the most recent, none of which ever got used, because I don't want them hanging around in the UI"